src/Symfony/Component/PropertyAccess/Tests/Fixtures/TestClassTypedProperty.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
should be removed and replaces by the license header, see other fixture files

Reply:
Fixed